### PR TITLE
Astrology Update

### PR DESCRIPTION
**Pull request by jandersson**
I added some command line args for astrology. I think it would be cool to add more features to the script and make them callable via arguments (e.g. `;astrology predict self athletics`). There was an issue with the rtr-expire flag causing the script to never cast RtR.

### CODE REVIEW

File: common-arcana.lic [hunk 1]
```diff
@@ -92,7 +92,7 @@ module DRCA
     when /^Your target pattern dissipates/, /^You can't cast that at yourself/, /^You need to specify a body part to consume/
       fput('release spell')
     when /^At the ritual's peak, your prophetic connection blooms a thousand-fold/
-      Flags.reset('rtr-expire')
+      Flags.add('rtr-expire', get_data('spells').spell_data['Read the Ripples']['expire'])
```
Comment (rpherbig) [inline common-arcana.lic]: Why is the Flag added here and in `astrology.lic`?

File: astrology.lic [hunk 2]
```diff
@@ -10,6 +10,12 @@ class Astrology
 
   def initialize
     settings = get_settings
+    arg_definitions = [
+	    [{name: 'train', regex: /train/i, description: 'Trains astrology'}],
```
Comment (rpherbig) [inline astrology.lic]: I would expect train to be the default behavior if no args are supplied. What do you think?

Comment (jandersson) [inline astrology.lic]: What is the way to supply a default argument?

Comment (jandersson) [inline astrology.lic]: I thought so too, i couldn't find any way to incorporate a default argument

Comment (rpherbig) [inline astrology.lic]:
```
arg_definitions = [
	    [],
            [{name: 'rtr', ...}]
]
```

File: common-arcana.lic (hunk 1 above)
Comment (jandersson) [inline common-arcana.lic]: I was having some issue with the flag being raised. I don't actually know what was causing it. The problem was the RtR expire flag would never be raised when the expire message came. It looks like adding it is the same as resetting it.

Comment (jandersson) [inline common-arcana.lic]: I will set this back to `reset` and post some logs

**Comment by jandersson** [comment]
OK Disregard the issue mentioned, using `Flags.reset` works fine in common-arcana

**Pull request merged**